mxcube3/routes/qutils.py

```diff
@@ -43,10 +44,14 @@ def node_index(node):
     elif node.get_parent():
         sample_model = node.get_parent().get_parent()
         sample = sample_model.loc_str
-        task_groups = sample_model.get_children();
+        task_groups = sample_model.get_children()
         group_list = [group.get_children() for group in task_groups]
         tlist = [task for task_list in group_list for task in task_list]
-        index = tlist.index(node)
+        # there is probably a better fix...
```

Comment:
As you state, I'm not sure that this is a fix. 

node_index(node) gets the index of node that is in the queue, it simply does not work on nodes that are not in the queue. So the question should be why is it called on nodes that are not in the queue. This try except basically only hides the problem, well we see it because there is the print but there is a risk of missing that. Maybe returning -1 or None for as index would be better.

But the question is really why are we trying to get the index in the queue of something that is not in the queue ?